packages/protocol/contracts/layer2/based/LibEIP1559.sol

```diff
@@ -52,7 +53,7 @@ library Lib1559Math {
         returns (uint64)
     {
         if (_gasTarget == 0) {
-            return _newGasTarget;
+            return _gasExcess;
```

Comment:
Shall we open a new PR to change this gas setting instead of in the cleanup PR?
But it shouldn't get into this conditional judgment.

Reply:
reverted this change